server/graphql/general/src/mutations/log.rs

```diff
@@ -0,0 +1,60 @@
+use async_graphql::*;
+use graphql_core::standard_graphql_error::validate_auth;
+use graphql_core::ContextExt;
+use service::{
+    auth::{Resource, ResourceAccessRequest},
+    settings::Level,
+};
+
+#[derive(Enum, Copy, Clone, PartialEq, Eq, Debug)]
+pub enum LogLevel {
+    Error,
+    Warn,
+    Info,
+    Debug,
+    Trace,
+}
+
+#[derive(InputObject)]
+#[graphql(name = "UpsertLogLevelInput")]
+pub struct LogLevelInput {
+    pub level: LogLevel,
+}
+
+#[derive(SimpleObject)]
+pub struct UpsertLogLevelResponse {
+    pub success_message: String,
+}
+
+pub fn upsert_log_level(
+    ctx: &Context<'_>,
+    store_id: String,
+    input: LogLevelInput,
+) -> Result<UpsertLogLevelResponse> {
+    validate_auth(
+        ctx,
+        &ResourceAccessRequest {
+            resource: Resource::ServerAdmin,
```

Comment:
Couldn't find a permission to update any log information... so have used ServerAdmin since log level can only be changed from Admin page and probably don't want to allow it to be changed by just anyone...

Reply:
sounds good to me